docker/lib/dependabot/docker/update_checker.rb

```diff
@@ -43,9 +43,9 @@ module Dependabot
   module Docker
     class UpdateChecker < Dependabot::UpdateCheckers::Base
       VERSION_REGEX =
-        /v?(?<version>[0-9]+(?:(?:\.[_a-z0-9]+)|(?:-(?:kb)?[0-9]+))*)/i
-      VERSION_WITH_SFX = /^#{VERSION_REGEX}(?<suffix>-[a-z0-9.\-]+)?$/i
-      VERSION_WITH_PFX = /^(?<prefix>[a-z0-9.\-]+-)?#{VERSION_REGEX}$/i
+        /v?(?<version>[0-9]+(?:\.[_a-z0-9]+|-(?:kb)?[0-9]+)*)/i
```

Comment:
```suggestion
      VERSION_REGEX = /v?(?<version>[0-9]+(?:\.[_a-z0-9]+|-(?:kb)?[0-9]+)*)/i
```

No need for the newline now that it's shorter, and the newline is making it more painful to review than it needs to be...